eo/Dockerfile

```diff
@@ -1,3 +1,16 @@
-FROM birdhouse/pavics-jupyter-base:0.0.2
+FROM birdhouse/pavics-jupyter-base:0.1.0
 
-RUN echo "test" > /eo-img-test
\ No newline at end of file
+# must update as root, because of a permission error with pip
+USER root
+
+COPY environment.yml /environment.yml
+
+# update env "birdy"
+# use umask 0000 so that the files for the updated environment are usable by the user for the jupyter-conda-extension
+RUN umask 0000 && conda env update -f /environment.yml
+
+# specify user because of problem running start-notebook.sh when being root
+USER jenkins
+
+# start notebook in conda environment to have working jupyter extensions
+CMD ["conda", "run", "-n", "birdy", "/usr/local/bin/start-notebook.sh"]
```

Comment:
This CMD already in the base image so no need to repeat it unless you changed it.

Reply:
True, I will remove it!